src/main/java/success/planfit/dto/response/PlaceDetailResponseDto.java

```diff
@@ -0,0 +1,63 @@
+package success.planfit.dto.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import success.planfit.domain.course.SpaceType;
+
+@ToString
+@Setter
+@Getter
+public class PlaceDetailResponseDto {
+
+    @JsonProperty("id")
+    private String googlePlacesIdentifier;
+
+    // 내부 클래스로 구현
+    @JsonProperty("displayName")
+    private DisplayName spaceName;
```

Comment:
```suggestion
    private String displayName;
```
`DisplayName`을 `String` 대신 내부 클래스로 구현하신 이유가 무엇인가요?
그를 통해 얻을 수 있는 이점이 있었다면 공유해주시면 감사하겠습니다!

Reply:
네 맞습니다! JSON으로 구현되어있는 String을 객체로 맵핑하기 위해 조금은 코드가 더럽지만 내부 클래스로 구현했습니다 더 좋은 방법을 찾게 된다면 추후에 수정해나가도록하겠습니다!

Reply:
`PlaceDetailResponseDto`는 구글 API의 요청을 받아오는 용도로만 사용하고, 클라이언트 측에 전달할 DTO는 따로 만드는 것은 어떻게 생각하시나요?

Reply:
좋습니다! 구현해놓도록 하겠습니다!